examples/with-prisma-data-provider/toolpad/prisma.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
In general we don't add eslint disable comments in examples, but turn off the rule in an eslint override.

Reason is that eslint is not used in the example, so it shouldn't contain eslint comments after I download it.